[2024-06-18T07:10:02.478+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T07:10:02.501+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: alerting_skipped.check_previous_run_status scheduled__2024-06-18T07:05:00+00:00 [queued]>
[2024-06-18T07:10:02.509+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: alerting_skipped.check_previous_run_status scheduled__2024-06-18T07:05:00+00:00 [queued]>
[2024-06-18T07:10:02.509+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-18T07:10:02.524+0000] {taskinstance.py:2330} INFO - Executing <Task(BranchPythonOperator): check_previous_run_status> on 2024-06-18 07:05:00+00:00
[2024-06-18T07:10:02.529+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1529) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-18T07:10:02.530+0000] {standard_task_runner.py:63} INFO - Started process 1530 to run task
[2024-06-18T07:10:02.530+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'alerting_skipped', 'check_previous_run_status', 'scheduled__2024-06-18T07:05:00+00:00', '--job-id', '427', '--raw', '--subdir', 'DAGS_FOLDER/alerting_skipped.py', '--cfg-path', '/tmp/tmp_20fv4a1']
[2024-06-18T07:10:02.531+0000] {standard_task_runner.py:91} INFO - Job 427: Subtask check_previous_run_status
[2024-06-18T07:10:02.571+0000] {task_command.py:426} INFO - Running <TaskInstance: alerting_skipped.check_previous_run_status scheduled__2024-06-18T07:05:00+00:00 [running]> on host fa5c5d972e05
[2024-06-18T07:10:02.657+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='alerting_skipped' AIRFLOW_CTX_TASK_ID='check_previous_run_status' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T07:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T07:05:00+00:00'
[2024-06-18T07:10:02.657+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T07:10:02.670+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-06-18T07:10:02.670+0000] {helper_functions.py:13} ERROR - Failed to run alerting_skipped with task check_last_run_status: This task is skipped on purpose
[2024-06-18T07:10:02.671+0000] {python.py:237} INFO - Done. Returned value was: start_run
[2024-06-18T07:10:02.671+0000] {branch.py:36} INFO - Branch into start_run
[2024-06-18T07:10:02.671+0000] {skipmixin.py:178} INFO - Following branch start_run
[2024-06-18T07:10:02.686+0000] {skipmixin.py:246} INFO - Skipping tasks [('skip_run', -1)]
[2024-06-18T07:10:02.710+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T07:10:02.728+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=alerting_skipped, task_id=check_previous_run_status, run_id=scheduled__2024-06-18T07:05:00+00:00, execution_date=20240618T070500, start_date=20240618T071002, end_date=20240618T071002
[2024-06-18T07:10:02.743+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-18T07:10:02.776+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-18T07:10:02.778+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
